lib/carin_for_blue_button_test_kit/generated/v1.1.0/explanation_of_benefit/metadata.yml

```diff
@@ -20,26 +20,42 @@
 - :names:
   - _id
   :expectation: SHALL
+  :names_not_must_support_or_mandatory: []
+  :must_support_or_mandatory: true
 - :names:
   - patient
   :expectation: SHALL
+  :names_not_must_support_or_mandatory: []
+  :must_support_or_mandatory: true
 - :names:
   - _lastUpdated
   :expectation: SHALL
+  :names_not_must_support_or_mandatory: []
+  :must_support_or_mandatory: true
 - :names:
   - type
   :expectation: SHALL
+  :names_not_must_support_or_mandatory: []
+  :must_support_or_mandatory: true
 - :names:
   - identifier
   :expectation: SHALL
+  :names_not_must_support_or_mandatory: []
+  :must_support_or_mandatory: true
 - :names:
   - service-date
   :expectation: SHALL
+  :names_not_must_support_or_mandatory: []
+  :must_support_or_mandatory: true
 :search_definitions:
   :_id:
-    :paths: []
-    :full_paths: []
+    :paths:
+    - id
+    :full_paths:
+    - Resource.id
```

Comment:
Why are the full_paths for id and lastUpdated getting set as Resource.id rather than ExplanationOfBenefit.id? Below, the patient search param has a full_path of ExplanationOfBenefit.patient, but it is different for id and lastUpdated. I believe I see that you found a way to handle this below, but I figured I would ask and see if there is any way to get it so it has the resource name instead of Resource.id